runtime/common/src/migrations.rs

```diff
@@ -102,6 +105,78 @@ where
 			DemocracyPalletName,
 			<Runtime as frame_system::Config>::DbWeight,
 		>::post_upgrade(_state);
+
+		Ok(())
+	}
+}
+
+pub struct RemoveCollectivesAddresses<Runtime>(pub PhantomData<Runtime>);
+impl<Runtime> Migration for RemoveCollectivesAddresses<Runtime>
+where
+	Runtime: frame_system::Config,
+{
+	fn friendly_name(&self) -> &str {
+		"MM_RemoveCollectivesAddresses"
+	}
+
+	fn migrate(&self, _available_weight: Weight) -> Weight {
+		log::info!("Removing gov v1 collective addresses storage");
+
+		// CouncilCollective: d59b9be6f0a7187ca6630c1d0a9bb045
+		let prefix = hex_literal::hex!("d59b9be6f0a7187ca6630c1d0a9bb045");
+		let result = clear_prefix(&prefix, Some(10), None);
+		log::info!("Removed {} CouncilCollective keys", result.unique);
+
+		// record how many records have been deleted
+		let mut writes = result.unique;
+
+		// TechCommitteeCollective: a06bfb73a86f8f98d5c5dc14e20e8a03
+		let prefix = hex_literal::hex!("a06bfb73a86f8f98d5c5dc14e20e8a03");
+		let result = clear_prefix(&prefix, Some(10), None);
```

Comment:
Are we sure that all runtimes have 10 or less keys? It is safe to have an higher value here, e.g `100`

Reply:
there are currently 5 members for each collective for all the networks, 10 is a sufficient number